pkiinit/option/generate.go

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright (c) 2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions and limitations under
+// the License.
+//
+// SPDX-License-Identifier: Apache-2.0'
+//
+package option
+
+import "fmt"
+
+// Generate option....
+func Generate() func(*PkiInitOption) (exitCode, error) {
```

Comment:
More code was written than necessary to finish the user story.  US4404 only asked that the "help" option be supported.

Reply:
this is just the boiler plate template code. not really implementation yet